source/Halibut.Tests/TestServices/AsyncSyncCompat/AdaptToTestCase.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Reflection;
+using System.Threading;
+using Halibut.Tests.Support;
+
+namespace Halibut.Tests.TestServices.AsyncSyncCompat
+{
+    public class AdaptToTestCase
```

Comment:
Maybe a hint this is about Async and Sync Adaption in the class name

Reply:
Done